patch/kconfig-inclusions

```diff
@@ -49,6 +49,13 @@ CONFIG_I2C_MUX_REG=m
 CONFIG_I2C_MUX_PCA954x=m
 CONFIG_I2C_MUX_MLXCPLD=m
 CONFIG_I2C_MLXCPLD=m
+CONFIG_I2C_DESIGNWARE_PLATFORM=m
+CONFIG_ACPI=y
+CONFIG_IOSF_MBI=y
+CONFIG_ARCH_SUPPORTS_ACPI=y
+CONFIG_I2C_DESIGNWARE_BAYTRAIL=y
+CONFIG_I2C_DESIGNWARE_CORE=m
+CONFIG_I2C_DESIGNWARE_PCI=m
```

Comment:
Please extend the commit message, explaining why the configuration needs to be changed. (What devices did not work for example, and do work now?)

Reply:
Added